fiftyone/utils/ultralytics.py

```diff
@@ -437,6 +435,68 @@ def predict_all(self, args):
         return self._format_predictions(predictions)
 
 
+class FiftyOneRTDETRModelConfig(FiftyOneYOLOModelConfig):
+    """Configuration for a :class:`FiftyOneRTDETRModel`.
+
+    Args:
+        model (None): an ``ultralytics.RTDETR`` model to use
+        model_name (None): the name of an ``ultralytics.RTDETR`` model to load
+        model_path (None): the path to an ``ultralytics.RTDETR`` model checkpoint
+    """
+
+    def __init__(self, d):
```

Comment:
nit: you could just write `pass` here; no need to actually define the `__init__` 🤓